nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryFulltextQuery.java

```diff
@@ -272,6 +272,7 @@ public void testFulltext() throws Exception {
         DocumentModel file1 = session.getDocument(new PathRef("/testfolder1/testfile1"));
         DocumentModel file2 = session.getDocument(new PathRef("/testfolder1/testfile2"));
         DocumentModel file3 = session.getDocument(new PathRef("/testfolder1/testfile3"));
+        // DocumentModel file3v1 = session.getLastDocumentVersion(file3.getRef())
```

Comment:
To remove ?

Reply:
I'm keeping it on purpose as documentation to explain that there's a v1 (unused) because later we also get the v2.